tests/test_format_unifier.py

```diff
@@ -0,0 +1,19 @@
+from utils.format_unifier import FormatUnifier
+from utils.tika_client import TikaClient
+from helpers import TEST_DATA_PATH
+import pytest
+
+format_unifier = FormatUnifier()
```

Comment:
To trzeba przenieść do `pytest.fixture`. Zobacz w `conftest.py` jak to się robi. Wtedy jak w argumencie testu podasz, to się wykona